src/main/resources/react/src/pages/UserInfo/tab_pages/my_activity/myActivity.js

```diff
@@ -0,0 +1,126 @@
+import React from 'react';
+
+import lot3 from '../../../../img/cards/lot3.jpg';
+import mommy from '../../../../img/mama_1_04232631 1.png';
+import TitleBigBlue from '../../../../components/title_Big_Blue';
+import ProductCard from '../../../../components/Card';
+
+import './myActyvity.scss';
+
+
+const MyActivity = () => {
+	const isFavorite = true;
+	return (
+		<div>
+			<TitleBigBlue whatClass={ 'incoming__replies-text' } text={ 'Входящие ответы' }/>
+			<div className={ 'block-of-cards' }>
+				<ProductCard
+					city={ 'Харьков' }
+					text={ `Велосипед ну очень куртой. просто не реально крутой для девочки 5 лет` }
+					picture={ lot3 }
+					inboxMessage={ 12 }
+					avatar={ '' }
+				/>
+				<ProductCard
```

Comment:
Duplication of logic. Make new function that will return `<ProductCard ...` and you pass all needed parameters into it.